src/js/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import { Box } from '../Box';
+import { Text } from '../Text';
+import { validEmail, getMd5 } from '../../utils';
+
+const Avatar = ({
+  align = 'center',
+  children,
+  email,
```

Comment:
i am a little sceptical for this. adding another dependency (bundle size) for something that not everybody will use (actually not sure who would use this). plus this is something the user could do on their own by providing a background how they want it to be whether its gravatar or any other similar service.

Reply:
Good comment. I was hoping generating an MD5 hash wouldn't be too heavy. If it is, I agree with removing this optimization.

Reply:
well bundlesize wise its not that heavy, 3.3kb minified (well sounds heavy for just an md5 actually) but feature wise as well its a maybe something not  alot of people would use to justify this 3.3kb.
just my opinion not sure it makes sense :)

Reply:
Good comments. Let me see if I can find a way to md5-encrypt locally on grommet rather than using a packaged. otherwise, I'll remove the email logic.

Reply:
Is this one lighter? https://preview.npmjs.com/package/md5

Reply:
[md5]( https://preview.npmjs.com/package/md5) is 30% lighter and stands on 2.2kB.
[pure-md5](https://www.npmjs.com/package/pure-md5) is even lighter than that, and stands on 1.9kB.    

Adding a new package, no matter its size, is still a commitment from our end, and its purpose      
of a single use-case that wasn't requested yet isn't too appealing to me.

I'm wondering if we should go the direction of adding a package or let the users that use Avatar + email do it on their end and maybe wait to see if anyone requests this feature in the future.

Reply:
Agreed with waiting to see where the demand is before adding md5.

Reply:
Thanks, removed `email` prop for now.